src/staging/gatsby-config.js

```diff
@@ -98,7 +98,18 @@ module.exports = {
     {
       resolve: `gatsby-plugin-robots-txt`,
       options: {
-        policy: [{userAgent: '*', allow: '/'}]
+        resolveEnv: () => process.env.ENV_SUFFIX,
+        env: {
+          'dev': {
+            policy: [{userAgent: '*', disallow: '/'}]
```

Comment:
`allow`  and `disallow` can both be a string or an array.  
https://github.com/itgalaxy/generate-robotstxt#usage

Can you please use the array format for these, so it will be self-documenting for future devs to know this option is available?